src/io/internal/pipelines/image/ReadDICOM/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ find_package(ITK REQUIRED
   COMPONENTS ITKCommon
     ITKGDCM
     ITKIOGDCM
+    ITKDCMTK
```

Comment:
@jadh4v can we remove this? CI should then pass.

Reply:
Done